tests/integration/component.go

```diff
@@ -712,4 +713,72 @@ func componentTests(args ...string) {
 			Expect(stdError).To(ContainSubstring("quantities must match the regular expression"))
 		})
 	})
+
+	Context("Creating component using symlink", func() {
+		JustBeforeEach(func() {
+			project = helper.CreateRandProject()
+			originalDir = helper.Getwd()
+			helper.Chdir(filepath.Dir(context))
```

Comment:
Unnecessarily changing the dir, right ? Without changing the dir it should work.